### DISCUSSION TITLE
Documentation coverage for Android

### DISCUSSION THREAD

**Discussion by LoredanaZdranc** [MobileNativeFoundation/discussions#119] [General]
Hey everyone! I am currently looking to identify a tool that can calculate the KPI for documentation coverage in Android. 
All code coverage tools I identified refer to test coverage, not to documentation coverage. 

On IOS there are few, like Jazzy or xcov, but I didn't identify one for Android.

Is there such a tool? Or can nothing help us in this case?

**Comment by 8vqjx80b** [comment]
Vaikka Dokka generoi HTML-sivuja, voit kirjoittaa yksinkertaisen skriptin, joka lukee Dokkan tuottamat JSON- tai XML-mallit ja vertaa niitä projektin koko lähdekoodirakenteeseen. Kun ajat Detektin CI-putkessa, saat XML-raportin, josta [mammona-kasino.net](https://mammona-kasino.net/) voit laskea dokumentoimattomien metodien määrän suhteessa kokonaismäärään. Tämä on selkein KPI, jota voit seurata ajan saatossa.
